src/view/welcome.js

```diff
@@ -29,8 +24,12 @@ const styles = StyleSheet.create({
 
 const WelcomeView = () => (
   <Background image="textured-bg" style={styles.background}>
-    <Icon image="lightning-bolt" style={styles.bolt} />
-    <Icon image="lightning-word" style={styles.name} />
+    <View style={styles.boltWrapper}>
```

Comment:
can a `style` prop be passed into an `Svg` component to avoid using these wrappers?

Reply:
Yeah, I tried that. Unfortunately the root `Svg` nodes are mapped to `svg` in the browser and those can't have a style attribute. They can on mobile however. So unfortunately the only cross-platform workaround is wrapping in a View 😕